pkg/client/kubelet.go

```diff
@@ -135,7 +136,43 @@ func (c *HTTPKubeletClient) GetPodStatus(host, podNamespace, podID string) (api.
 }
 
 func (c *HTTPKubeletClient) HealthCheck(host string) (probe.Result, error) {
-	return httprobe.DoHTTPProbe(fmt.Sprintf("%s/healthz", c.url(host)), c.Client)
```

Comment:
Why not just write a non-200 error code from the healthcheck and detect this condition from the kubelet?